scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -102,6 +102,16 @@
     env_aware=True,
 )
 
+define_config_option(
+    __monitor__,
+    "first_time_old_container_window",
```

Comment:
I don't think "old" container is really descriptive here.  I would suggest calling this option "initial_stopped_container_collection_window".  I know it's wordy, but I'd definitely like to use the word "stopped" instead of "old".